CHANGELOG.md

```diff
@@ -105,111 +109,122 @@
 * Removed old error message that's superseded by `requires_dependencies`.
 * Removes using `hi_res` as the default strategy value for `partition_via_api` and `partition_multiple_via_api`
 
+## 0.8.2-dev3
+
+### Enhancements
+
+*   set the file's current position to the beginning after reading the file in `convert_to_bytes`
```

Comment:
Please take out these extra tabs that got added in the changelog

Reply:
Only change in this file should be the update to the version and the new changelog entry for the fix

Reply:
This and the next line still have the additional whitespace